src/mlpack/core/cv/metrics/average_strategy.hpp

```diff
@@ -21,6 +21,14 @@ namespace cv {
  *
  * The "Binary" strategy means binary classification is going to be used, and
  * there is no need to average.
+ *
+ * The "Micro" strategy is used for multiclass classification, in this strategy
+ * TP(True Positive), TN(True Negative) and FN(False Negative) all are counted
+ * globally for calculating metrics.
+ *
+ * The "Macro" strategy is also used for multiclass classification, in this
+ * metric is calculated for each label separately and then an unweighted
+ * average is taken.
```

Comment:
Honestly speaking I still find this to be somewhat incorrect and not very descriptive.  I'm happy to merge the other changes you have made though, so if you want, we can drop this attempt and merge the rest.

Reply:
@rcurtin I am good if you drop this attempt and merge the rest.